[rmlui] Build error on x64-mingw-dynamic

There is no file as suggested by: Copy issue body from /myworkdir/vcpkg_installed/vcpkg/issue_body.md

Hello, I'm not quite familiar with this community format, and the suggested file doesn't exist so I will just report:

Steps to reproduce:
- Using Windows 11
- Using mingw
- On an empty project with just 
  - `vcpkg new --application`
  - `vcpkg add port rmlui`
- With env variable: `VCPKG_DEFAULT_HOST_TRIPLET` : `x64-mingw-dynamic`
- Use `vcpkg install`

Error extract:
```powershell
-- Applying patch add-itlib-and-robin-hood.patch
-- Applying patch skip-custom-find-modules.patch
-- Using source at <path to vcpkg>/vcpkg/buildtrees/rmlui/src/6.1-fbf6dfc3b5.clean
-- Configuring x64-mingw-dynamic
-- Building x64-mingw-dynamic-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: "C:/Program Files/CMake/bin/cmake.exe" --build . --config Debug --target install -- -v -j9
    Working Directory: <again my path to vcpkg>/vcpkg/buildtrees/rmlui/x64-mingw-dynamic-dbg
    See logs for more information
```

Upon reading the logs we find some errors with the robin-hood.h from the patch. Notably:
```bash
 note: 'uint32_t' is defined in header '<cstdint>'; this is probably fixable by adding '#include <cstdint>'
   45 | #include <limits>
  +++ |+#include <cstdint>
   46 | #include <memory> // only to support hash of smart pointers
```

--- a/ports/robin-hood-hashing/fix-missing-stdint.patch
+++ b/ports/robin-hood-hashing/fix-missing-stdint.patch
@@ -1,0 +1,12 @@
+diff --git a/src/include/robin_hood.h b/src/include/robin_hood.h
+index b4e0fbc..405e67d 100644
+--- a/src/include/robin_hood.h
++++ b/src/include/robin_hood.h
+@@ -40,6 +40,7 @@
+ 
+ #include <algorithm>
+ #include <cstdlib>
++#include <cstdint>
+ #include <cstring>
+ #include <functional>
+ #include <limits>

--- a/ports/robin-hood-hashing/portfile.cmake
+++ b/ports/robin-hood-hashing/portfile.cmake
@@ -4,6 +4,8 @@ vcpkg_from_github(
 	REF 3.11.5
 	SHA512 5f73e089b1e8ec41a9bedded22bc64a789d3a3d04873a2ad9f8cc2970797a473b0f4d3436c2324b3ced85a0d983998a75b1dfaf2b7f3b77235b29806ff2fd489
 	HEAD_REF master
+    PATCHES
+        fix-missing-stdint.patch
 )
 
 vcpkg_cmake_configure(

--- a/ports/robin-hood-hashing/portfile.cmake
+++ b/ports/robin-hood-hashing/portfile.cmake
@@ -2,7 +2,7 @@ vcpkg_from_github(
 	OUT_SOURCE_PATH SOURCE_PATH
 	REPO martinus/robin-hood-hashing
 	REF 3.11.5
-	SHA512 0
+	SHA512 5f73e089b1e8ec41a9bedded22bc64a789d3a3d04873a2ad9f8cc2970797a473b0f4d3436c2324b3ced85a0d983998a75b1dfaf2b7f3b77235b29806ff2fd489
 	HEAD_REF master
     PATCHES
         fix-missing-stdint.patch

--- a/ports/robin-hood-hashing/portfile.cmake
+++ b/ports/robin-hood-hashing/portfile.cmake
@@ -2,7 +2,7 @@ vcpkg_from_github(
 	OUT_SOURCE_PATH SOURCE_PATH
 	REPO martinus/robin-hood-hashing
 	REF 3.11.5
-	SHA512 5f73e089b1e8ec41a9bedded22bc64a789d3a3d04873a2ad9f8cc2970797a473b0f4d3436c2324b3ced85a0d983998a75b1dfaf2b7f3b77235b29806ff2fd489
+	SHA512 0
 	HEAD_REF master
     PATCHES
         fix-missing-stdint.patch

--- a/ports/robin-hood-hashing/vcpkg.json
+++ b/ports/robin-hood-hashing/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "robin-hood-hashing",
   "version": "3.11.5",
-  "port-version": 1,
+  "port-version": 2,
   "description": "Fast & memory efficient hashtable based on robin hood hashing for C++11/14/17/20",
   "homepage": "https://github.com/martinus/robin-hood-hashing",
   "license": "MIT",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -8542,7 +8542,7 @@
     },
     "robin-hood-hashing": {
       "baseline": "3.11.5",
-      "port-version": 1
+      "port-version": 2
     },
     "robin-map": {
       "baseline": "1.4.0",

--- a/versions/r-/robin-hood-hashing.json
+++ b/versions/r-/robin-hood-hashing.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "1d1cf4ecfda26cc7953c491aa40ffd5fd6ddfc00",
+      "git-tree": "1a989171742275cc32060ed37157eeb0efcdc739",
       "version": "3.11.5",
       "port-version": 2
     },

--- a/versions/r-/robin-hood-hashing.json
+++ b/versions/r-/robin-hood-hashing.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "1d1cf4ecfda26cc7953c491aa40ffd5fd6ddfc00",
+      "version": "3.11.5",
+      "port-version": 2
+    },
+    {
       "git-tree": "25ce100807060e0a7cf0c0f8f0bed52bca9a4ea8",
       "version": "3.11.5",
       "port-version": 1

